llvm/test/Transforms/InstCombine/AArch64/sve-intrinsic-ld1ro.ll

```diff
@@ -0,0 +1,37 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt -S -mattr=+sve,+f64mm -passes=instcombine -aarch64-sve-vector-bits-min=512 -aarch64-sve-vector-bits-max=512 < %s | FileCheck %s
```

Comment:
As above `aarch64-sve-vector-bits-max` is really for internal code generation testing.  You likely want to mirror the logic used when specifying `-msve-vector-bit` which adds `vscale_range` attributes to the IR.

Reply:
Thanks, add attribute `vscale_range(4,4)` for IR, then remove the option `aarch64-sve-vector-bits-max`